packages/react/src/PolkadotWalletsContext.tsx

```diff
@@ -18,19 +17,17 @@ const PolkadotWalletsContext = createContext<PolkadotWalletsContextProps>({
 
 export const useWallets = () => useContext(PolkadotWalletsContext);
 
-export const PolkadotWalletsContextProvider = ({
-  children,
-  walletAggregator,
-  initialWaitMs = 5 /* the default is set to 5ms to give extensions enough lead time to inject their providers */,
-}: PolkadotWalletsContextProviderProps) => {
+export const PolkadotWalletsContextProvider = ({ children, walletAggregator }: PolkadotWalletsContextProviderProps) => {
   const [wallets, setWallets] = useState<BaseWallet[] | undefined>();
 
   useEffect(() => {
-    const timeoutId = setTimeout(() => {
-      setWallets(walletAggregator.getWallets());
-    }, initialWaitMs);
-    return () => clearTimeout(timeoutId);
-  }, [walletAggregator]);
+    const getWallets = async () => {
```

Comment:
We still want to keep this. The observation was that PJS is slow injecting its provider. So the InjectedWeb3 will be available but PJS might be missing.